Makefile

```diff
@@ -11,6 +11,8 @@ GO=GOOS=linux GO111MODULE=on GOFLAGS=-mod=vendor go
 GO_BUILD_RECIPE=$(GO) build -o $(OUT_DIR)/$(PACKAGE_BIN) -ldflags '-X $(PACKAGE)/version.Version=$(REV)' $(PACKAGE_MAIN)
 GOFMT_CHECK=$(shell find . -not \( \( -wholename './.*' -o -wholename '*/vendor/*' \) -prune \) -name '*.go' | sort -u | xargs gofmt -s -l)
 REV=$(shell git describe --long --tags --match='v*' --always --dirty)
+TOOLS_BIN_DIR=$(OUT_DIR)/bin
```

Comment:
I see that some of the binaries in NTO are under OUT_DIR , maybe we should still keep them there?   @jmencak WDYT?

Reply:
Either keep them there or change to `$(OUT_DIR)/bin` and adjust the Containerfiles.

Reply:
Ok. Changed to OUT_DIR